src/connectors/rating-menu/connectRatingMenu.ts

```diff
@@ -16,7 +23,16 @@ const $$type = 'ais.ratingMenu';
 const MAX_VALUES_PER_FACET_API_LIMIT = 1000;
 const STEP = 1;
 
-const createSendEvent = ({
+type SendEvent = (...args: [InsightsEvent] | [string, string, string?]) => void;
+
+type CreateSendEvent = (createSendEventArgs: {
+  instantSearchInstance: InstantSearch;
+  helper: AlgoliaSearchHelper;
+  getRefinedStar: () => number | number[] | undefined;
+  attribute: string;
+}) => SendEvent;
+
+const createSendEvent: CreateSendEvent = ({
```

Comment:
these changes possibly need to be rebased, I thought this was merged separately

Reply:
Done (mistake on my side, this PR was intended to have `feat/ts-ratingmenu` as base and not `master`)